service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksLogSettingsStep.java

```diff
@@ -22,7 +22,7 @@ public class CreateAksLogSettingsStep extends BaseResourceCreateStep {
   private static final Logger logger = LoggerFactory.getLogger(CreateAksLogSettingsStep.class);
 
   // 365 is the max value; this is the limitation of Azure
-  private static final int RETENTION_DAYS = 365;
+  private static final int RETENTION_DAYS = 0;
```

Comment:
this is indefinite log retention, isn't it? Is that what we really want?

Reply:
I do not know!  @aherbst-broad I know this got the tests passing but do we need a better permanent solution?

Reply:
Ah, the comment on that commit is "… and rely on the LTS storage retetion lifecycle policies" so I'm guessing that's fine